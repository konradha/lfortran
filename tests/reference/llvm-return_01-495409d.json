{
    "basename": "llvm-return_01-495409d",
    "cmd": "lfortran --no-color --show-llvm {infile} -o {outfile}",
    "infile": "tests/../integration_tests/return_01.f90",
    "infile_hash": "04561922c972a04f60f447b912c77c581f4ea58c64f6cbce4fc1ae2e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "llvm-return_01-495409d.stdout",
    "stdout_hash": "192d1098fe399f4630340fbd35e480fb78ba62ed236e8ca903ca7af2",
    "stderr": "llvm-return_01-495409d.stderr",
    "stderr_hash": "3c7370e49583f13017d1500c533fc16ec67eefb9732878d66392964c",
    "returncode": 0
}