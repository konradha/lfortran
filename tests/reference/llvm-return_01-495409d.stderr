style suggestion: Use '>' instead of '.gt.'
 --> tests/../integration_tests/return_01.f90:8:15
  |
8 |         if (i .GT. 5) then
  |               ^^^^ help: write this as '>'
